### PR TITLE
[FIX] sale: fix wrong keyboard shortcut for 'create invoice'

### PR DESCRIPTION
**Pull request by mathysPaul**
Resolves the conflict where two shortcuts (alt-c for 'new' and
'create invoice') existed, but only 'new' could be triggered.
Updates the shortcut for 'create invoice' to alt-i, ensuring
both shortcuts are functional and accessible.

task-3628572

### CODE REVIEW

**Review by Demesmaeker — APPROVED** [review]
Hello :duck: 

lgtm, could you just add the task number in the commit? Thank you ^^

File: addons/sale/views/sale_order_views.xml [hunk 1]
```diff
@@ -250,7 +250,7 @@
                     type="action" class="btn-primary" data-hotkey="c"
```
Comment (dalaOdoo) [inline addons/sale/views/sale_order_views.xml:250]:
```suggestion
                    type="action" class="btn-primary" data-hotkey="i"
```

**Comment by Feyensv** [comment]
> lgtm, could you just add the task number in the commit? Thank you ^^

Still not handled, also the two commits can be squashed together.

**Review by Feyensv — APPROVED** [review]
@robodoo r+

**Pull request closed without merging**